api/namespace.go

```diff
@@ -70,8 +70,15 @@ type Namespace struct {
 	Name        string
 	Description string
 	Quota       string
-	CreateIndex uint64
-	ModifyIndex uint64
+	// TODO: how to encode properly?
+	Capabilities *NamespaceCapabilities
```

Comment:
I think we just need this:

```suggestion
	Capabilities *NamespaceCapabilities `hcl:"capabilities,block"`
```